pom.xml

```diff
@@ -28,6 +28,11 @@
         <artifactId>gson</artifactId>
         <version>2.8.6</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
```

Comment:
Why?
We have appengine version 1.9.80 right below this

Reply:
Oh good catch thank you